app/Main.hs

```diff
@@ -43,15 +38,23 @@ import CardanoNode (
 -- Hydra auction CLI imports
 import CLI.Actions (CliAction, handleCliAction)
 import CLI.Parsers (
-  CliInput (..),
+  CliInput (InteractivePrompt, Watch),
+  PromptOptions (..),
   getCliInput,
   parseCliAction,
  )
+import CLI.Watch (
+  watchAuction,
+ )
 
 main :: IO ()
 main = do
-  MkCliInput {cliVerbosity, cliActor, cliNodeSocket, cliNetworkId} <- getCliInput
+  ci <- getCliInput
+  handleCliInput ci
 
+handleCliInput :: CliInput -> IO ()
+handleCliInput (Watch auctionName) = watchAuction auctionName
+handleCliInput (InteractivePrompt MkPromptOptions {..}) = do
```

Comment:
Just for visibility I want to repeat what I said on the other review, namely that I think for records with many fields this is not very nice as it makes looking up the record fields very hard.

Reply:
While I don't personally agree with the hate `RecordWildcards` gets, it is worth mentioning that we explicitly mention not to use it in the styleguide https://github.com/mlabs-haskell/hydra-auction/blob/staging/doc/contributing.md#other-guidelines yet have it enabled in several files through pragmas 😅

Reply:
Part of the issue, is we say: don't use it, except if you really need to. which is too vague to be enforceable, maybe we can come up with a concrete number of fields for which we allow it (or just ban it completely!)